File: app/src/main/java/com/lightcyclesoftware/photoscodeexample/PhotoApplication.java
package com.lightcyclesoftware.photoscodeexample;


import android.app.Application;

import com.facebook.drawee.backends.pipeline.Fresco;

public class PhotoApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        Fresco.initialize(this);
    }
}

File: app/src/main/java/com/lightcyclesoftware/photoscodeexample/application/PhotoApplication.kt
package com.lightcyclesoftware.photoscodeexample.application

import android.app.Application
import com.facebook.drawee.backends.pipeline.Fresco

/**
 * Created by Edward on 2/4/2018.
 */

class PhotoApplication: Application() {
    override fun onCreate() {
        super.onCreate()
        Fresco.initialize(this)
    }
}